share/doc/diana.1.in
.\"                                      Hey, EMACS: -*- nroff -*-
.Dd Oct 17, 2017
.Os
.Dt DIANA 1
.Sh NAME
.Nm diana.bin-@PVERSION@
.Nd MET Norway meteorological visualisation software
.Sh SYNOPSIS
.Nm
[\fB\-h\fR]
[\fB\-v\fR]
[\fB\-L\fR \fIlog4cpp.properties\fR]
[\fB\-s\fR \fIsetupfile\fR]
[\fB\-I\fR \fIinstancename\fR]
.Sh DESCRIPTION
.Nm
is a meteorological visualisation software developed by MET Norway,
SMHI, and others.

Please use
.Nm
\fB-h\fR for help. It is generally recommended to use a script to
start Diana.

.Sh SEE ALSO
https://github.com/metno/diana, https://wiki.met.no/diana/start

.Sh AUTHOR
Diana is written at Met.no, the Norwegian Meteorological Institute,
and SMHI, the Swedish Meteorological and Hydrological Institute.  This
manual page was written by Alexander B\(:urger
<alexander.buerger@met.no> in October 2017.


debian/changelog
diana (3.28.0-ubuntu1) lucid; urgency=low

  * using fimex 0.37.1
  * using metlibs_trunk
  * compiled without profet support
	
 -- Lisbeth Bergholt <lisbethb@met.no>  Thu, 29 Mar 2012 21:42:51 +0200

diana (3.27.4-ubuntu1) lucid; urgency=low

  * Observations - bugfix criterias

 -- Lisbeth Bergholt <lisbethb@met.no>  Thu, 01 Mar 2012 18:55:51 +0100

diana (3.27.3-ubuntu1) lucid; urgency=low

  * HQC - bugfix

 -- Lisbeth Bergholt <lisbethb@met.no>  Mon, 27 Feb 2012 14:31:58 +0100

diana (3.27.2-ubuntu1) lucid; urgency=low

  * New EPS parameter - cluster representative
  * Bugfix profet: computed fields (like FF) were not
	changed when the original fields (u,v) were edited. 

 -- Lisbeth Bergholt <lisbethb@met.no>  Tue, 14 Feb 2012 20:32:42 +0100

diana (3.27.1-ubuntu1) lucid; urgency=low

  * Minor fixes

 -- Lisbeth Bergholt <lisbethb@met.no>  Thu, 09 Feb 2012 14:48:14 +0100

diana (3.27.0-ubuntu1) lucid; urgency=low

  * Using metlibs_4.3
  * Using libfimex-0.35
  * Using WDB/fimex
  * Reading from KDVH

 -- Lisbeth Bergholt <lisbethb@met.no>  Thu, 26 Jan 2012 09:42:51 +0100

diana (3.26.4-ubuntu3) lucid; urgency=low

  * bdiana: default canvas changed to QGLPixelBuffer

 -- Lisbeth Bergholt <lisbethb@pc2982.pc.met.no>  Mon, 28 Nov 2011 09:42:43 +0100

diana (3.26.4-ubuntu2) lucid; urgency=low

  * using +towgs in all proj strings except old mitiff files.
  * Using metlibs_4.2
  * Using libfimex-0.33-1

 -- Lisbeth Bergholt <lisbethb@vpn-116-99.met.no>  Sat, 26 Nov 2011 16:42:14 +0100

diana (3.26.4-ubuntu1) lucid; urgency=low

  * Bugfix:DifferenceField - plot string was not parsed correctly
  * Using metlibs_trunk (by mistake)

 -- Lisbeth Bergholt <lisbethb@met.no>  Tue, 22 Nov 2011 13:54:05 +0100

diana (3.26.3-ubuntu1) lucid; urgency=low

  * Bugfix: Ascii-observations - only observations from one file were used
  * Using metlibs_trunk (by mistake)

 -- Lisbeth Bergholt <lisbethb@.met.no>  Mon, 21 Nov 2011 13:30:46 +0100

diana (3.26.2-ubuntu1) lucid; urgency=low

  * Bugfix: mark measurements/trajectory positions

 -- Lisbeth Bergholt <lisbethb@met.no>  Sun, 09 Oct 2011 17:28:50 +0200

diana (3.26.1-ubuntu2) lucid; urgency=low

  * Bugfix bdiana: vertical levels

 -- Lisbeth Bergholt <lisbethb@met.no>  Tue, 27 Sep 2011 14:09:07 +0200

diana (3.26.1-ubuntu1) lucid; urgency=low

  * Bugfix bdiana: translation from plot name to field name

 -- Lisbeth Bergholt <lisbethb@met.no>  Tue, 27 Sep 2011 12:52:03 +0200

diana (3.26.0-ubuntu2) lucid; urgency=low

  * Projections: datum fix removed

 -- Lisbeth Bergholt <lisbethb@met.no>  Mon, 26 Sep 2011 10:39:04 +0200

diana (3.26.0-ubuntu1) lucid; urgency=low

  * Using metlibs_4.2
  * Using libfimex-0.31-2
  * New profet function
  * Improved fimex reading
  * Vertical crossections: vt+omega / vt+w fixed
  * Bufr: skip time variables with missing valaues 
  * Bugfix:projections

 -- Lisbeth Bergholt <lisbethb@met.no>  Fri, 23 Sep 2011 15:38:14 +0200

diana (3.25.5-ubuntu2) lucid; urgency=low

  * Bugfix: bdiana did not find fieldTimes

 -- Lisbeth Bergholt <lisbethb@met.no>  Mon, 04 Jul 2011 12:41:27 +0200

diana (3.25.5-ubuntu1) lucid; urgency=low

  * Bugfix: Labels from ObsAscii files
  * Using metlibs_4.1
  * Using libfimex-dev 0.29

 -- Lisbeth Bergholt <lisbethb@met.no>  Mon, 27 Jun 2011 09:45:07 +0200

diana (3.25.4-ubuntu2) lucid; urgency=low

  * Bugfix was not included in last version

 -- Lisbeth Bergholt <lisbethb@met.no>  Wed, 22 Jun 2011 08:52:03 +0200

diana (3.25.4-ubuntu1) lucid; urgency=low

  * Bugfix: Memory leak in StationPlot
  * PostScript fix: Always set font-specification

 -- Lisbeth Bergholt <lisbethb@met.no>  Tue, 21 Jun 2011 14:53:45 +0200

diana (3.25.3-ubuntu1) lucid; urgency=low

  * More bugfix: profet

 -- Lisbeth Bergholt <lisbethb@met.no>  Mon, 20 Jun 2011 13:30:54 +0200

diana (3.25.2-ubuntu2) lucid; urgency=low

  * Bugfix case insensitivity

 -- Lisbeth Bergholt <lisbethb@met.no>  Fri, 17 Jun 2011 14:54:36 +0200

diana (3.25.2-ubuntu1) lucid; urgency=low

  * Bugfix xylimit
  * Reverted fix: model and field are now case insensitive

 -- Lisbeth Bergholt <lisbethb@met.no>  Fri, 17 Jun 2011 08:28:03 +0200

diana (3.25.1-ubuntu1) lucid; urgency=low

  * Field: model and field are now case sensitive

 -- Lisbeth Bergholt <lisbethb@met.no>  Thu, 16 Jun 2011 11:26:45 +0200

diana (3.25.0-ubuntu4) lucid; urgency=low

  * Using metlibs_4.1
  * Using libfimex-dev 0.28-2

 -- Lisbeth Bergholt <lisbethb@met.no>  Wed, 15 Jun 2011 10:06:40 +0200

diana (3.25.0-ubuntu3) lucid; urgency=low

  * Using metlibs_4.1
  * Using libfimex-dev 0.28

 -- Lisbeth Bergholt <lisbethb@met.no>  Tue, 10 May 2011 08:31:50 +0200

diana (3.25.0-ubuntu2) lucid; urgency=low

  * Geotiff support enabled
  * Using metlibs_4.1

 -- Lisbeth Bergholt <lisbethb@met.no>  Thu, 05 May 2011 12:02:30 +0200

diana (3.25.0-ubuntu1) lucid; urgency=low

  * New field reader using fimex
  * Using metlibs_2011-01-28

 -- Lisbeth Bergholt <lisbethb@met.no>  Fri, 28 Jan 2011 12:47:24 +0100

diana (3.24.0-ubuntu2) lucid; urgency=low

  * Bugfix: enable/disable ObjectPlots
  * Using metlibs_2011-01-13

 -- Lisbeth Bergholt <lisbethb@met.no>  Mon, 24 Jan 2011 11:01:52 +0100

diana (3.24.0-ubuntu1) lucid; urgency=low

  * Using metlibs_2011-01-13
  * new version of profet

 -- Lisbeth Bergholt <lisbethb@met.no>  Fri, 21 Jan 2011 13:26:08 +0100

diana (3.23.3-ubuntu1) lucid; urgency=low

  * Bugfix parsing logfile
  * Using metlibs_2010-11-15

 -- Lisbeth Bergholt <lisbethb@vpn-117-63.met.no>  Mon, 06 Dec 2010 09:00:58 +0100

diana (3.23.2-ubuntu1) lucid; urgency=low

  * Bugfix obsplot: pressure tendency, temperature precision, metar clouds 
  * Bugfix editing: Snow_showers -> Snow showers

 -- Lisbeth Bergholt <lisbethb@met.no>  Mon, 29 Nov 2010 13:22:29 +0100

diana (3.23.1-ubuntu6) lucid; urgency=low

  * Added export LANG=en_US in diana.in

 -- Lisbeth Bergholt <lisbethb@met.no>  Tue, 23 Nov 2010 14:00:45 +0100


diana.sh
#!/bin/sh
set -x

input=$0
dianadir="/usr/local"

export LD_LIBRARY_PATH=/metno/local/lib/mesa:$LD_LIBRARY_PATH

export DIANADIR=$dianadir

OPSYS=`uname -s`
debugger="gdb"
SUBNET=`/sbin/ifconfig eth0 | grep "inet addr" | cut -d. -f3`
ulimit -c 100000000000

case $SUBNET in
    20)    region="FOU" ;;
    36|40) region="VV" ;;
    48)    region="VNN" ;;
    16|18|24|26|56|90|104|50) region="VA"  ;;
    *)                         region="VTK" ;;
esac

home=$HOME/.diana
test -d $home || mkdir $home
test -d $home/work  || mkdir $home/work
test -e $home/diana.setup || cp $dianadir/etc/diana/diana.setup-${region} $home/diana.setup
cd $home

rm core* 1>/dev/null 2>&1
test -s dbxresult  && mv -f dbxresult dbxresult.old
test -s mail.core  && mv -f mail.core mail.core.old

# remove old lpr print files, left after failure...
find . -name "prt_????-??-??_??:??:??.ps" -mtime +1 -exec rm {} \;

tstart=`date`

$dianadir/bin/diana.bin -s $home/diana.setup -style cleanlooks

tstop=`date`

corefound="no"
for corefile in core*
do
    test -f $corefile || continue
    test -s $corefile || continue
    corefound="yes"
done

if [ $corefound = "yes" ] ; then

echo "USER $USER"     >mail.core
echo "-------------" >>mail.core
uname -a             >>mail.core
echo "-------------" >>mail.core
echo "START $tstart" >>mail.core
echo "STOP  $tstop"  >>mail.core
echo "Diana version (diana.news):" >>mail.core 
cat diana.news       >>mail.core
echo "-------------" >>mail.core
pwd                  >>mail.core
ls -ltr core*        >>mail.core

for corefile in core*
do

test -f $corefile || continue
test -s $corefile || continue

$debugger $dianadir/bin/diana.bin $corefile 1>dbxresult 2>&1 <<ENDDBX
where 
y
y
y
n
q
ENDDBX

echo "===============================================" >>mail.core
cat dbxresult        >>mail.core

rm $corefile

done

echo "===============================================" >>mail.core
hinv                 >>mail.core

adr="audun.christoffersen@met.no helen.korsmo@met.no lisbeth.bergholt@met.no"
Mail -s "DIANA CRASH" $adr < mail.core

fi

exit
